CHANGELOG.md

```diff
@@ -2,6 +2,121 @@
 
 Changelogs for versions not listed here can be found at https://github.com/DataDog/dd-trace-py/releases
 
+---
+
+## 2.9.0
+
+### New Features
+
+- LLM Observability: This introduces the LLM Observability SDK, which enhances the observability of Python-based LLM applications. See the [LLM Observability Overview](https://docs.datadoghq.com/tracing/llm_observability/) or the [SDK documentation](https://docs.datadoghq.com/tracing/llm_observability/sdk) for more information about this feature.
+- ASM: This introduces full support for exploit prevention in the python tracer.  
+  - LFI (via standard API open)
+  - SSRF (via standard API urllib or third party requests)
+
+  with monitoring and blocking feature, telemetry and span metrics reports.
+
+- opentelemetry: Adds support for span events.
+
+- tracing: Ensures the following OpenTelemetry environment variables are mapped to an equivalent Datadog configuration (datadog environment variables taking precedence in cases where both are configured):
+
+      OTEL_SERVICE_NAME -> DD_SERVICE
+      OTEL_LOG_LEVEL -> DD_TRACE_DEBUG
+      OTEL_PROPAGATORS -> DD_TRACE_PROPAGATION_STYLE
+      OTEL_TRACES_SAMPLER -> DD_TRACE_SAMPLE_RATE
+      OTEL_TRACES_EXPORTER -> DD_TRACE_ENABLED
+      OTEL_METRICS_EXPORTER -> DD_RUNTIME_METRICS_ENABLED
+      OTEL_LOGS_EXPORTER -> none
```

Comment:
I missed this in code review. `OTEL_LOGS_EXPORTER` does not map to a Datadog environment variable. Having none here feels off. 

```suggestion
```